x/ecocredit/server/core/create_batch.go

```diff
@@ -51,7 +51,7 @@ func (k Keeper) CreateBatch(ctx context.Context, req *core.MsgCreateBatch) (*cor
 		return nil, err
 	}
 
-	startDate, endDate := timestamppb.New(req.StartDate.UTC()), timestamppb.New(req.EndDate.UTC())
+	startDate, endDate := timestamppb.New(*req.StartDate), timestamppb.New(*req.EndDate)
```

Comment:
I missed this the first time around... I don't think we want to store local times if the user provides a local time. I think we always want to store the UTC time.

Reply:
cc @clevinson

Reply:
whoops, nice catch - fixed here https://github.com/regen-network/regen-ledger/pull/1160/commits/524e2f5cf0b7f930f05e299575a778ff7a1769d5